sonaranalyzer-dotnet/build/build-dev.ps1

```diff
@@ -0,0 +1,21 @@
+. .\build-utils.ps1
+
+$mode = "dev"
+
+Set-Version -buildNumber 5
+
+Begin-Analysis $mode

```

Comment:
Not sure it is useful to run analysis in dev. Especially when you are pushing to next.